### ISSUE TITLE
Why DVBTTelevizor relies entirely on DvbDriver?

### ISSUE THREAD

**Issue by Xf0ur** [petrj/DVBTTelevizor#37]
Hi,
I compiled kernel for my DVB-T card thinking that it will work with DVBTTelevizor.
Sadly, it didn't 😕
That means it depends on DvbDriver only.
Is there anyway to change that?

**Comment by Xf0ur** [comment]
Hi,
I meant that my DVB-T card is now supported by the kernel after i enabled its modules then compiled it.
Why i need any driver if all linux apps like Tvheadend, VDR, Kaffeine  ... etc can recognize it?
I was asking if DVBT Televizor can recognize it too like the above mentioned apps.

**Comment by petrj** [comment]
No. DVBT Televizor is designed to work only with DvbDriver. DvbDriver communicates with HW layer and provides a TCP/IP interface for DVBT Televizor. Theoretically can be the driver replaced by another with same TCP/IP interface.
If I had more time in the future I'd like to write my own driver, because I can communicate with SDR HW (see my https://github.com/petrj/RTL-SDR-Receiver), but it's not relevant now.

**Comment by Xf0ur** [comment]
[https://www.youtube.com/watch?v=HYGPCSn73hk](url)

If you check that youtube video.
Although the DVB-T card isn't supported by DvbDriver it works with Tvheadend.
So no need for DvbDriver or any other driver.
It's easier to modify DVBT Televizor in order to communicate with the DVB card directly.
Isn't it?

**Comment by petrj** [comment]
Tvheadend, VLC, Kaffeine, mplayer and other SW communicate with HW directly, DVB-T televizor don't. As I wrote in previous comment: If I had more time, I'd like to change it, but I'm busy now.

**Comment by Xf0ur** [comment]
Okay, i hope you will be have some free time soon.
Btw, by doing that DVB-T televizor will support a lot of DVB-T devices that are supported under Linux.
My last question though.
What about minisatip?
If i compiled it and run it on my Android device, can DVB-T televizor communicate with it?
Surely after making some changes to it 1st.

**Comment by Xf0ur** [comment]
Hello,
Can you verify whether ProgTV+ works with your DVB-T card without DvbDriver?

**Comment by Xf0ur** [comment]
ProgTV for Android works without DvbDriver as well.
See this video
https://www.youtube.com/watch?v=TAVTCTGVIws
Now your turn 😄
